doc/Standaard-inrichting-GitHub-GitLab.md

```diff
@@ -120,6 +134,8 @@ Indien gewenst kunnen er, net als voor de verschillende soorten issues, template
 Een eenduidige werkwijze vertaalt zich ook naar de op de issues en PR's toe te passen labels.
 Natuurlijk vult elke repository de behoefte van zijn gebruikers zo goed mogelijk in wat kan betekenen dat er een aantal specifiek in de betreffende repository worden aangemaakt. Over het algemeen kennen de repositories echter wel een aantal labels die het gemeen heeft met andere repositories en idealiter hebben deze over de verschillende repositories eenzelfde kleur. In de onderstaande tabel zijn een aantal van deze labels opgenomen met de daarbij horende hexadecimale waarde van de kleur.
 
+OPMERKING: Dit is hoe er in de HaalCentraal omgeving wordt gewerkt. Standaard github en versiebeheer methode is het werken met releases en tags. Bij het Zaakgerichten werken project worden tags juist gebruikt om onderwerpen/thema's/epics aan te geven. Onderstaande tabel zou ik niet overnemen als standaard werkwijze.
```

Comment:
Onderstaande lijst is voor het labelen van issues. Zo kun je dus zichtbaar maken welke user stories je in v2.0 wil meenemen. Je kunt hiermee ook de issue lijst filteren zodat alle bugs worden getoond.

Wij gebruiken dit mechanisme ook in GitHub pages. Zo wordt er automatisch een md bestand gegenereerd met daarin gesorteerd op het versie label de bijbehorende user stories ([zie hier](https://vng-realisatie.github.io/Haal-Centraal-BRP-bevragen/user-stories-prod).)

Jullie gebruiken de epic labels dus waar wij de versie labels gebruiken.
Ik kan me dus voorstellen dat jullie niet de versie labels willen gebruiken maar ik ben er van overtuigd dat er labels zijn die we delen. Onderstaande lijst wil ik hier niet helemaal verwijderen omdat dit voor mij handig is ihkv HC maar wellicht kunnen we er een kolom aan toevoegen waarin we aangeven of het een ZGW, HC of algemeen label is.

Reply:
De epic labels zijn in het verleden gebruikt, momenteel gebruik ik de github milestones functionaliteit. Labels gaan over een onderwerp maar niet over een release. Bijvoorbeeld "documentatie", "vraag" of een speciifieke API waar een issue over gaat.

Dit is precies wat ik bedoelde met werk op de hals halen. Bij ZGW worden releases op een heel andere manier, met github tooling, gedefinieerd. Dat past niet op elkaar. Geeft niets, dan doen we dat anders.

Reply:
**Voorstel:**
vervang de tekst in deze sectie door:

> Een eenduidige werkwijze vertaalt zich ook naar een aantal op de issues en PR's toe te passen labels.
> 
> Natuurlijk vult elke repository de behoefte van zijn gebruikers zo goed mogelijk in wat kan betekenen dat er een aantal specifieke label per repository worden aangemaakt. Over het algemeen kennen de repositories echter ook een aantal labels die het gemeen heeft met andere repositories en idealiter hebben deze over de verschillende repositories eenzelfde kleur. In de onderstaande tabel zijn deze labels opgenomen met de daarbij horende hexadecimale waarde van de kleur.
> 
> | Labelnaam | Beschrijving | Kleur |
| --- | --- | --- |
| bug | Fout in de code.  | #d73a4a |
| user story | Business vraag van een gemeentelijke afnemer. | #82f2cb |
| Documentation | Verbeteringen of toevoegingen aan de documentatie. | #0075ca |
| wontfix | Dit wordt niet opgelost. | #ffffff |
> 
> M.b.t. versiebeheer kan gekozen worden voor verschillende manieren van werken. Issues kunnen geördend worden m.b.v. milestones of m.b.v. labels. Indien dit gebeurd m.b.v. labels dan voldoen deze labelnamen aan de volgende syntax 'v[m].[n]'. Daarbij staat 'm' voor de major releasenummer en 'n' voor het min releasenummer, bijv. 'v1.0' of 'v2.3'.
> Standaardisatie van deze labels is van belang voor het script dat voor de GitHub Pages omgeving de releasenotes genereert. 

Indien we aan dit document ook beschrijvingen op willen nemen die gerelateerd zijn aan GitHub Pages dan zou je als laatste zin nog op kunnen nemen:

> Zie voor meer informatie m.b.t. deze scripts de sectie 'GitLab Pages'.

Reply:
@michielverhoef Ik heb een kopje 'Milestones' toegevoegd. Wellicht kan jij aangeven of daar dingen standaard ingericht kunnen worden. Bij het kopje 'Standaard labels' heb ik aangegeven dat dit alleen van toepassing is als er geen gebruik wordt gemaakt van Milestones.